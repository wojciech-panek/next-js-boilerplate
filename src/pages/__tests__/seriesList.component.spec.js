import React from 'react';
import { shallow } from 'enzyme';

import SeriesList from '../seriesList';


describe('SeriesList: Page', () => {
  const defaultProps = {
  };

  const component = (props) => (
    <SeriesList {...defaultProps} {...props} />
  );

  it('should render correctly', () => {
    const wrapper = shallow(component());
    global.expect(wrapper).toMatchSnapshot();
  });
});
