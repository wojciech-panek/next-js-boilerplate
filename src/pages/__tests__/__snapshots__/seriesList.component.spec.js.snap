// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeriesList: Page should render SeriesList root 1`] = `
<div>
  Series List
</div>
`;
